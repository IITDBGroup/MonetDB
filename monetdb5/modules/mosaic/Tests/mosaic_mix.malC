# OID list compressions

o := 19531025:int;

b:= bat.new(:int);
bat.append(b,0);
bat.append(b,1);
bat.append(b,2);
bat.append(b,3);
bat.append(b,5);
bat.append(b,7);
bat.append(b,70);
bat.append(b,188);
bat.append(b,190);
bat.append(b,192);
bat.append(b,9999);
bat.append(b,50);
bat.append(b,49);
bat.append(b,50);
bat.append(b,50);
bat.append(b,48);
bat.append(b,1003);
bat.append(b,1002);
bat.append(b,1001);
bat.append(b,1000);
bat.append(b,95);
bat.append(b,99);
bat.append(b,96);
bat.append(b,98);
bat.append(b,97);
bat.append(b,94);
bat.append(b,90);
bat.append(b,93);
bat.append(b,91);
bat.append(b,92);
bat.append(b,1140);
bat.append(b,1130);
bat.append(b,1120);
bat.append(b,1110);
bat.append(b,1100);
bat.append(b,1250);
bat.append(b,1260);
bat.append(b,1270);
bat.append(b,1280);
bat.append(b,1290);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,o);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
bat.append(b,9);
b:= bat.append(b,b);

io.print(b);
mosaic.compress(b);
b:= mosaic.decompress(b);
io.print(b);
