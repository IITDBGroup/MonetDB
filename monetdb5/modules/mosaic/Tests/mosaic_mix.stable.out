stdout of test 'mosaic_mix` in directory 'monetdb5/modules/mosaic` itself:


# 20:55:33 >  
# 20:55:33 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=37588" "--set" "mapi_usock=/var/tmp/mtest-15513/.s.monetdb.37588" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/export/scratch1/mk/mosaic//Linux/var/MonetDB/mTests_monetdb5_modules_mal" "mosaic_mix.mal"
# 20:55:33 >  

# MonetDB 5 server v11.18.0
# This is an unreleased version
# Serving database 'mTests_monetdb5_modules_mal', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 15.591 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2014 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://vienna.ins.cwi.nl:37588/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-15513/.s.monetdb.37588
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded
# MonetDB/R   module loaded

# 11:50:48 >  
# 11:50:48 >  "mclient" "-lmal" "-ftest" "-tnone" "-Eutf-8" "--host=/var/tmp/mtest-30332" "--port=39586"
# 11:50:48 >  

# 15:01:26 >  
# 15:01:26 >  "mclient" "-lmal" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-15809" "--port=37660"
# 15:01:26 >  

#io.print(b);
#--------------------------#
# h	t  # name
# void	int  # type
#--------------------------#
[ 0@0,	0  ]
[ 1@0,	1  ]
[ 2@0,	2  ]
[ 3@0,	3  ]
[ 4@0,	5  ]
[ 5@0,	7  ]
[ 6@0,	70  ]
[ 7@0,	188  ]
[ 8@0,	190  ]
[ 9@0,	192  ]
[ 10@0,	9999  ]
[ 11@0,	50  ]
[ 12@0,	49  ]
[ 13@0,	50  ]
[ 14@0,	50  ]
[ 15@0,	48  ]
[ 16@0,	1003  ]
[ 17@0,	1002  ]
[ 18@0,	1001  ]
[ 19@0,	1000  ]
[ 20@0,	95  ]
[ 21@0,	99  ]
[ 22@0,	96  ]
[ 23@0,	98  ]
[ 24@0,	97  ]
[ 25@0,	94  ]
[ 26@0,	90  ]
[ 27@0,	93  ]
[ 28@0,	91  ]
[ 29@0,	92  ]
[ 30@0,	1140  ]
[ 31@0,	1130  ]
[ 32@0,	1120  ]
[ 33@0,	1110  ]
[ 34@0,	1100  ]
[ 35@0,	1250  ]
[ 36@0,	1260  ]
[ 37@0,	1270  ]
[ 38@0,	1280  ]
[ 39@0,	1290  ]
[ 40@0,	19531025  ]
[ 41@0,	19531025  ]
[ 42@0,	19531025  ]
[ 43@0,	19531025  ]
[ 44@0,	19531025  ]
[ 45@0,	19531025  ]
[ 46@0,	19531025  ]
[ 47@0,	19531025  ]
[ 48@0,	19531025  ]
[ 49@0,	19531025  ]
[ 50@0,	19531025  ]
[ 51@0,	19531025  ]
[ 52@0,	19531025  ]
[ 53@0,	19531025  ]
[ 54@0,	19531025  ]
[ 55@0,	19531025  ]
[ 56@0,	19531025  ]
[ 57@0,	19531025  ]
[ 58@0,	19531025  ]
[ 59@0,	19531025  ]
[ 60@0,	9  ]
[ 61@0,	9  ]
[ 62@0,	9  ]
[ 63@0,	9  ]
[ 64@0,	9  ]
[ 65@0,	9  ]
[ 66@0,	9  ]
[ 67@0,	9  ]
[ 68@0,	9  ]
[ 69@0,	9  ]
[ 70@0,	9  ]
[ 71@0,	9  ]
[ 72@0,	9  ]
[ 73@0,	9  ]
[ 74@0,	9  ]
[ 75@0,	9  ]
[ 76@0,	9  ]
[ 77@0,	9  ]
[ 78@0,	9  ]
[ 79@0,	9  ]
[ 80@0,	9  ]
[ 81@0,	9  ]
[ 82@0,	9  ]
[ 83@0,	9  ]
[ 84@0,	9  ]
[ 85@0,	9  ]
[ 86@0,	9  ]
[ 87@0,	9  ]
[ 88@0,	9  ]
[ 89@0,	9  ]
[ 90@0,	0  ]
[ 91@0,	1  ]
[ 92@0,	2  ]
[ 93@0,	3  ]
[ 94@0,	5  ]
[ 95@0,	7  ]
[ 96@0,	70  ]
[ 97@0,	188  ]
[ 98@0,	190  ]
[ 99@0,	192  ]
[ 100@0,	9999  ]
[ 101@0,	50  ]
[ 102@0,	49  ]
[ 103@0,	50  ]
[ 104@0,	50  ]
[ 105@0,	48  ]
[ 106@0,	1003  ]
[ 107@0,	1002  ]
[ 108@0,	1001  ]
[ 109@0,	1000  ]
[ 110@0,	95  ]
[ 111@0,	99  ]
[ 112@0,	96  ]
[ 113@0,	98  ]
[ 114@0,	97  ]
[ 115@0,	94  ]
[ 116@0,	90  ]
[ 117@0,	93  ]
[ 118@0,	91  ]
[ 119@0,	92  ]
[ 120@0,	1140  ]
[ 121@0,	1130  ]
[ 122@0,	1120  ]
[ 123@0,	1110  ]
[ 124@0,	1100  ]
[ 125@0,	1250  ]
[ 126@0,	1260  ]
[ 127@0,	1270  ]
[ 128@0,	1280  ]
[ 129@0,	1290  ]
[ 130@0,	19531025  ]
[ 131@0,	19531025  ]
[ 132@0,	19531025  ]
[ 133@0,	19531025  ]
[ 134@0,	19531025  ]
[ 135@0,	19531025  ]
[ 136@0,	19531025  ]
[ 137@0,	19531025  ]
[ 138@0,	19531025  ]
[ 139@0,	19531025  ]
[ 140@0,	19531025  ]
[ 141@0,	19531025  ]
[ 142@0,	19531025  ]
[ 143@0,	19531025  ]
[ 144@0,	19531025  ]
[ 145@0,	19531025  ]
[ 146@0,	19531025  ]
[ 147@0,	19531025  ]
[ 148@0,	19531025  ]
[ 149@0,	19531025  ]
[ 150@0,	9  ]
[ 151@0,	9  ]
[ 152@0,	9  ]
[ 153@0,	9  ]
[ 154@0,	9  ]
[ 155@0,	9  ]
[ 156@0,	9  ]
[ 157@0,	9  ]
[ 158@0,	9  ]
[ 159@0,	9  ]
[ 160@0,	9  ]
[ 161@0,	9  ]
[ 162@0,	9  ]
[ 163@0,	9  ]
[ 164@0,	9  ]
[ 165@0,	9  ]
[ 166@0,	9  ]
[ 167@0,	9  ]
[ 168@0,	9  ]
[ 169@0,	9  ]
[ 170@0,	9  ]
[ 171@0,	9  ]
[ 172@0,	9  ]
[ 173@0,	9  ]
[ 174@0,	9  ]
[ 175@0,	9  ]
[ 176@0,	9  ]
[ 177@0,	9  ]
[ 178@0,	9  ]
[ 179@0,	9  ]
#none 1
#none 2
#none 3
#none 4
#none 5
#none 6
#none 7
#none 8
#none 9
#none 10
#none 11
#none 12
#none 13
#none 14
#none 15
#none 16
#none 17
#none 18
#none 19
#none 20
#none 21
#none 22
#none 23
#none 24
#none 25
#none 26
#none 27
#none 28
#none 29
#none 30
#none 31
#none 32
#none 33
#none 34
#none 35
#none 36
#none 37
#none 38
#none 39
#none 40
#none 41
#none 42
#none 43
#none 44
#none 45
#none 46
#none 47
#none 48
#none 49
#none 50
#none 51
#none 52
#none 53
#none 54
#none 55
#none 56
#none 57
#none 58
#none 59
#none 60
#none 61
#none 62
#none 63
#none 64
#none 65
#none 66
#none 67
#none 68
#none 69
#none 70
#none 71
#none 72
#none 73
#none 74
#none 75
#none 76
#none 77
#none 78
#none 79
#none 80
#none 81
#none 82
#none 83
#none 84
#none 85
#none 86
#none 87
#none 88
#none 89
#none 90
#none 91
#none 92
#none 93
#none 94
#none 95
#none 96
#none 97
#none 98
#none 99
#none 100
#none 101
#none 102
#none 103
#none 104
#none 105
#none 106
#none 107
#none 108
#none 109
#none 110
#none 111
#none 112
#none 113
#none 114
#none 115
#none 116
#none 117
#none 118
#none 119
#none 120
#none 121
#none 122
#none 123
#none 124
#none 125
#none 126
#none 127
#none 128
#none 129
#none 130
#none 131
#none 132
#none 133
#none 134
#none 135
#none 136
#none 137
#none 138
#none 139
#none 140
#none 141
#none 142
#none 143
#none 144
#none 145
#none 146
#none 147
#none 148
#none 149
#none 150
#none 151
#none 152
#none 153
#none 154
#none 155
#none 156
#none 157
#none 158
#none 159
#none 160
#none 161
#none 162
#none 163
#none 164
#none 165
#none 166
#none 167
#none 168
#none 169
#none 170
#none 171
#none 172
#none 173
#none 174
#none 175
#none 176
#none 177
#none 178
#none 179
#none 180
#--------------------------#
# h	t  # name
# void	int  # type
#--------------------------#
[ 0@0,	0  ]
[ 1@0,	1  ]
[ 2@0,	2  ]
[ 3@0,	3  ]
[ 4@0,	5  ]
[ 5@0,	7  ]
[ 6@0,	70  ]
[ 7@0,	188  ]
[ 8@0,	190  ]
[ 9@0,	192  ]
[ 10@0,	9999  ]
[ 11@0,	50  ]
[ 12@0,	49  ]
[ 13@0,	50  ]
[ 14@0,	50  ]
[ 15@0,	48  ]
[ 16@0,	1003  ]
[ 17@0,	1002  ]
[ 18@0,	1001  ]
[ 19@0,	1000  ]
[ 20@0,	95  ]
[ 21@0,	99  ]
[ 22@0,	96  ]
[ 23@0,	98  ]
[ 24@0,	97  ]
[ 25@0,	94  ]
[ 26@0,	90  ]
[ 27@0,	93  ]
[ 28@0,	91  ]
[ 29@0,	92  ]
[ 30@0,	1140  ]
[ 31@0,	1130  ]
[ 32@0,	1120  ]
[ 33@0,	1110  ]
[ 34@0,	1100  ]
[ 35@0,	1250  ]
[ 36@0,	1260  ]
[ 37@0,	1270  ]
[ 38@0,	1280  ]
[ 39@0,	1290  ]
[ 40@0,	19531025  ]
[ 41@0,	19531025  ]
[ 42@0,	19531025  ]
[ 43@0,	19531025  ]
[ 44@0,	19531025  ]
[ 45@0,	19531025  ]
[ 46@0,	19531025  ]
[ 47@0,	19531025  ]
[ 48@0,	19531025  ]
[ 49@0,	19531025  ]
[ 50@0,	19531025  ]
[ 51@0,	19531025  ]
[ 52@0,	19531025  ]
[ 53@0,	19531025  ]
[ 54@0,	19531025  ]
[ 55@0,	19531025  ]
[ 56@0,	19531025  ]
[ 57@0,	19531025  ]
[ 58@0,	19531025  ]
[ 59@0,	19531025  ]
[ 60@0,	9  ]
[ 61@0,	9  ]
[ 62@0,	9  ]
[ 63@0,	9  ]
[ 64@0,	9  ]
[ 65@0,	9  ]
[ 66@0,	9  ]
[ 67@0,	9  ]
[ 68@0,	9  ]
[ 69@0,	9  ]
[ 70@0,	9  ]
[ 71@0,	9  ]
[ 72@0,	9  ]
[ 73@0,	9  ]
[ 74@0,	9  ]
[ 75@0,	9  ]
[ 76@0,	9  ]
[ 77@0,	9  ]
[ 78@0,	9  ]
[ 79@0,	9  ]
[ 80@0,	9  ]
[ 81@0,	9  ]
[ 82@0,	9  ]
[ 83@0,	9  ]
[ 84@0,	9  ]
[ 85@0,	9  ]
[ 86@0,	9  ]
[ 87@0,	9  ]
[ 88@0,	9  ]
[ 89@0,	9  ]
[ 90@0,	0  ]
[ 91@0,	1  ]
[ 92@0,	2  ]
[ 93@0,	3  ]
[ 94@0,	5  ]
[ 95@0,	7  ]
[ 96@0,	70  ]
[ 97@0,	188  ]
[ 98@0,	190  ]
[ 99@0,	192  ]
[ 100@0,	9999  ]
[ 101@0,	50  ]
[ 102@0,	49  ]
[ 103@0,	50  ]
[ 104@0,	50  ]
[ 105@0,	48  ]
[ 106@0,	1003  ]
[ 107@0,	1002  ]
[ 108@0,	1001  ]
[ 109@0,	1000  ]
[ 110@0,	95  ]
[ 111@0,	99  ]
[ 112@0,	96  ]
[ 113@0,	98  ]
[ 114@0,	97  ]
[ 115@0,	94  ]
[ 116@0,	90  ]
[ 117@0,	93  ]
[ 118@0,	91  ]
[ 119@0,	92  ]
[ 120@0,	1140  ]
[ 121@0,	1130  ]
[ 122@0,	1120  ]
[ 123@0,	1110  ]
[ 124@0,	1100  ]
[ 125@0,	1250  ]
[ 126@0,	1260  ]
[ 127@0,	1270  ]
[ 128@0,	1280  ]
[ 129@0,	1290  ]
[ 130@0,	19531025  ]
[ 131@0,	19531025  ]
[ 132@0,	19531025  ]
[ 133@0,	19531025  ]
[ 134@0,	19531025  ]
[ 135@0,	19531025  ]
[ 136@0,	19531025  ]
[ 137@0,	19531025  ]
[ 138@0,	19531025  ]
[ 139@0,	19531025  ]
[ 140@0,	19531025  ]
[ 141@0,	19531025  ]
[ 142@0,	19531025  ]
[ 143@0,	19531025  ]
[ 144@0,	19531025  ]
[ 145@0,	19531025  ]
[ 146@0,	19531025  ]
[ 147@0,	19531025  ]
[ 148@0,	19531025  ]
[ 149@0,	19531025  ]
[ 150@0,	9  ]
[ 151@0,	9  ]
[ 152@0,	9  ]
[ 153@0,	9  ]
[ 154@0,	9  ]
[ 155@0,	9  ]
[ 156@0,	9  ]
[ 157@0,	9  ]
[ 158@0,	9  ]
[ 159@0,	9  ]
[ 160@0,	9  ]
[ 161@0,	9  ]
[ 162@0,	9  ]
[ 163@0,	9  ]
[ 164@0,	9  ]
[ 165@0,	9  ]
[ 166@0,	9  ]
[ 167@0,	9  ]
[ 168@0,	9  ]
[ 169@0,	9  ]
[ 170@0,	9  ]
[ 171@0,	9  ]
[ 172@0,	9  ]
[ 173@0,	9  ]
[ 174@0,	9  ]
[ 175@0,	9  ]
[ 176@0,	9  ]
[ 177@0,	9  ]
[ 178@0,	9  ]
[ 179@0,	9  ]

# 20:55:33 >  
# 20:55:33 >  "Done."
# 20:55:33 >  

